src/main/groovy/net/wasdev/wlp/gradle/plugins/Liberty.groovy

```diff
@@ -205,21 +208,71 @@ class Liberty implements Plugin<Project> {
             }
         }
         serverMap.remove('class')
+        serverMap.remove('outputDir')
 
         return ServerExtension.newInstance(serverMap)
     }
 
-    void setServersForTasks(Project project){
+    public static void checkEtcServerEnvProperties(Project project) {
+        Properties envProperties = new Properties()
+        //check etc/server.env and set liberty.outputDir
+        File serverEnvFile = new File(Liberty.getInstallDir(project), 'etc/server.env')
+        if (serverEnvFile.exists()) {
+            envProperties.load(new FileInputStream(serverEnvFile))
+            Liberty.setLibertyOutputDir(project, (String) envProperties.get("WLP_OUTPUT_DIR"))
+        }
+    }
+
+    public static void checkServerEnvProperties(Project project) {
+        Properties envProperties = new Properties()
```

Comment:
I think we only want to do this if thisServer.outputDir is null - not already set by the build file.